.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,52 @@
+name: Benchmarks
+
+on:
+  push:
+    branches: [main]
+  schedule:
+    - cron: "0 2 * * *"
+  workflow_dispatch:
+
+jobs:
+  run-benchmarks:
+    name: Run benchmarks
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+        python-version: ["3.8"]
```

Comment:
Will we be expanding this to also test against `macos-latest` and later Python versions, as in the other tests, in the future?

Reply:
I'm not sure the OS should be a variable in benchmarks. The key factor should be the delta version-on-version.